scss/components/_tooltips.scss

```diff
@@ -0,0 +1,48 @@
+// Tooltips
+//
+// Tooltips for maps
+//
+// Markup:
+// <div class="tooltip">
+//  <div class="tooltip__title">Michigan</div>
+//  <div class="tooltip__value">$100,000</div>
+// </div>
+//
+// Styleguide components.tooltips
+
+.tooltip {
```

Comment:
Should there be a max-width on them? They're appearing full width on the styleguide.